sanic/app.py

```diff
@@ -1027,8 +1027,10 @@ def run(
                 "stop_event will be removed from future versions.",
                 DeprecationWarning,
             )
-        # compatibility old access_log params
-        self.config.ACCESS_LOG = access_log
+        # if access_log is passed explicitly change config.ACCESS_LOG
+        if access_log is not None:
```

Comment:
Should we make sure that `access_log` is a boolean?

Reply:
I didn't saw any type checks in that method. I think that if somebody will pass a string or something in there he will see an error quite fast. I also like type hinting but it is almost not used in Sanic.

Reply:
Yes ... but something we need to fix. There is no time like the present :sunglasses: But I was suggesting one step further than just type hinting.

Reply:
Do you want to raise an Exception or just ignore the value if it is not a bool?

Reply:
Being that it is something that happens on startup, and should control an important feature, I think Exception is warranted.

(PS - Thanks for the work on this :beers: )